### PR TITLE
Add styling to project

### PR DESCRIPTION
**Pull request by Kwamay**
In this model, I.......

-[ ]  Added CSS styling to project to match Zeplin's design

### CODE REVIEW

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
.

File: src/component/Book.jsx [hunk 1]
```diff
@@ -8,27 +8,27 @@ const Book = (props) => {
   } = props;
   return (
     <div className="book-container">
-      <div>
-        <p>{category}</p>
-        <h3>{title}</h3>
-        <small>{author}</small>
-        <div>
-          <button type="button">Comments</button>
+      <div className="book-details">
```
Comment (tufoinnkuo10) [inline src/component/Book.jsx:11]: - [ ] GReat job so far. please kindly change the background color ` #fafafa` around the cards so it looks like the zeplin design.
- [ ] please your progress update buttons should shift inside a little as they are not supposed to be aligned with the user photo place holder and the add book button.
- [ ] note that the book cards have a different width with the main top bar holding the user photo and main title. please fix that so it follows the zeplin design. 
- [ ] please kindly align the add book button with the book cards and the user photo place holder. nice work
yours
Screenshot_48
original
Screenshot_49

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @Kwamay 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Gitflow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ Design well followed

Required Changes ♻️
Check the comments under the review
Optional suggestions
- [optional] Please kindly change your project title in the readme file so it portrays what you are working on.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by Lucas-Masaba — APPROVED** [review]
# Hello @Kwamay  👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

almost-there

## Highlights 🍾 
- Linter checks passing :heavy_check_mark: 
- Made changes as requested by the previous reviewer :+1: 

## Optional suggestions 💭 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**